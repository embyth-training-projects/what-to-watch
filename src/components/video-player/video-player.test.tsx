import * as React from "react";
import * as renderer from "react-test-renderer";

import ViderPlayer from "./video-player";

const videoData = {
  poster: `img/the-grand-budapest-hotel-poster.jpg`,
  preview: `https://upload.wikimedia.org/wikipedia/commons/7/72/Landwasserviadukt%2C_aerial_video.webm`,
};

it(`Should render correctly VideoPlayer`, () => {
  const tree = renderer
    .create(
        <ViderPlayer
          muted
          isPlaying={false}
          source={videoData.preview}
          poster={videoData.poster}
        />, {
          createNodeMock: () => {
            return {};
          }
        }
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
