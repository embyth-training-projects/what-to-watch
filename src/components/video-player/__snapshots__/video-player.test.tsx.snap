// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly VideoPlayer 1`] = `
<video
  height="175"
  width="280"
>
  Sorry, your browser doesn't support embedded videos.
</video>
`;
